src/viewport/index.ts

```diff
@@ -1,5 +1,27 @@
-export * from "./viewport-matcher.directive";
-export * from "./viewport-server-size.service";
-export * from "./viewport.model";
-export * from "./viewport.service";
-export * from "./viewport.util";
```

Comment:
changed wildcard to specific imports - however still need to go through and check what needs to be exported

Reply:
ideally leave as much as possible (so its less of a breaking)

just remove `isViewportConditionMatch` since that has a breaking and it should have been internal